setup.py

```diff
@@ -14,10 +17,7 @@
       packages=find_packages(exclude=['docs']),
       include_package_data=True,
       zip_safe=False,
-      install_requires=[
-          'click',
-          'PyYAML',
-      ],
+      install_requires=requirements,
```

Comment:
Why did this change? In general, it's better to have the requirements in `setup.py` without version pins. (You can put `-e .` in `requirements.txt` to have your package and its dependencies installed by things that look for that.)